### DISCUSSION TITLE
Mouse sensitivity on 32:9 monitor

### DISCUSSION THREAD

**Discussion by edmuff** [hrvach/deskhop#95] [Q&A]
Hi, first of, amazing project and fun build! I have Samsung G9 ultra mega wide monitor and my mouse sense just goes crazy when I connect it via the deskhop. Sensitivity side to side is very high but up and down is normal. Is this something I can change or am I missning something?

**Comment by hrvach** [comment]
Check user_config.h, there is a "Mouse Speed Factor" section with a few settings you can define for any output or direction... Make sure you do a full config wipe after changing this (shortcut in readme).

I'm trying to make this configurable in a more user-friendly way...
